recipes/pywaterml/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pywaterml" %}
+{% set version = "1.2.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/romer8/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 636118f005a451e9ef64dbdeb47a7e518a8f8684bb1fcd07d9607fe9473ff618
+
+build:
+
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```

Comment:
The `host` section really is just what's needed to run `setup.py`, not the full package. It looks like this should only need to be:
```yaml
- python >=3.6
- pip
- setuptools
```

Reply:
I changed it to only used the suggested dependecies in the host